### PR TITLE
Migrate database to PostgreSQL for better structuring and access from Grafana

### PR DESCRIPTION
**Pull request by LMNYX**
Will follow with approximate 10 minute downtime to migrate all the data and re-encode what 's left of new data.

### CODE REVIEW

**Comment by clevernt** [comment]
LGTM

**Comment by LMNYX** [comment]
> [LGTM](https://github.com/GoofyGoofsterClub/imagination-server/pull/47#issuecomment-2354182144)

doesn't look good to me yet.

File: Dockerfile [hunk 1]
```diff
@@ -10,6 +10,8 @@ COPY . .
 
 EXPOSE 3443
 
+RUN npm run build
```
Comment (LMNYX) [inline Dockerfile:13]: idk if it should stay for prod because it worked without it??? idk how

**Comment by LMNYX** [comment]
image

**Comment by LMNYX** [comment]
Revised

uwugraph

**Pull request merged**